lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -855,6 +855,41 @@ static void optimizeBatchNorm(Function *F) {
   } // For all nodes in the graph.
 }
 
+/// Check if the input of this method (vector of NodeValues) can be 
+/// legal input of CreateConcat function.
+/// \returns the dimension where the Concat will take place. If not 
+/// concat-able, return -1.
+size_t checkInputsForConcat(std::vector<NodeValue> inputs) {
```

Comment:
May be use a more descriptive name instead of checkInputsForConcat. E.g. getConactDimensionForInputs or something like that? But it is just a minor remark. I don't have a strong opinion here.

Reply:
Also, this function should be probably static, right?

Reply:
done. changed to static